tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -24,9 +24,6 @@ def __init__(self, marionette):
     def launch(self, launch_timeout=None):
         self.app = self.apps.launch(self.name, self.manifest_url, self.entry_point, launch_timeout=launch_timeout)
 
-    def wait_for_condition(self, method, timeout=None, message=None):
```

Comment:
I think 2 more calls of this function are missing, in:
* `test_dialer.py`
* `test_dialer_find_contact.py`